### PR TITLE
Filtros OK

### PR DESCRIPTION
**Pull request by JuanManu59**

### CODE REVIEW

File: cypress/page/products-list.page.ts [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 
-  public addToMyCart(): void {
-    cy.get(this.addCarBtn).click();
-  }
-
-  public proceedCheckout(): void {
+  public addTShirtToCart(productName: string): void {
+    this.findProductByName(productName).find(this.addCarBtn).click();
     cy.get(this.checkoutBtn).click();
+}
```
Comment (holgiosalos) [inline cypress/page/products-list.page.ts]: review the indentation

File: cypress/page/products-list.page.ts [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+
+  public findProductByName(productName: string): void {
```
Comment (holgiosalos) [inline cypress/page/products-list.page.ts]: this method can be private

File: cypress/page/products-list.page.ts (hunk 1 above)
Comment (JuanManu59) [reply cypress/page/products-list.page.ts]: resolved

File: cypress/page/products-list.page.ts (hunk 2 above)
Comment (JuanManu59) [reply cypress/page/products-list.page.ts]: resolved

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**